apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -700,6 +700,7 @@ public void downloadBlob(String container, String name, File destination, Execut
 
          Futures.getUnchecked(Futures.allAsList(results));
 
+         raf.close();
       } catch (IOException e) {
          // cleanup, attempt to delete large file
          if (raf != null) {
```

Comment:
Same here. If you close it in the finally block, there is no need to close it here too.